src/elfpy/strategies/single_lp.py

```diff
@@ -10,29 +10,14 @@
 
 
 class Policy(BasicPolicy):
-    """
-    simple LP
-    only has one LP open at a time
-    """
-
-    def __init__(self, market, rng, wallet_address, budget=1000):
-        """call basic policy init then add custom stuff"""
-        self.amount_to_lp = 100
-        super().__init__(
-            market=market,
-            rng=rng,
-            wallet_address=wallet_address,
-            budget=budget,
-        )
+    "single LP: only has one LP open at a time"
+    amount_to_lp = 100
```

Comment:
This is now a static element that belongs to the class. Before, the variable was an object member variable (belongs to `self`). The difference is whether you want the variable to exist even if the class is not instantiated. What you've done here is not necessarily _wrong_, but it is conveying the wrong message. We want `amount_to_lp` to be belonged by some instantiated user. e.g.
```
from elfpy.strategies.single_lp import Policy
alice = Policy()
alice.amount_to_lp = 100
bob = Policy()
bob.amount_to_lp = 1000
```

so in this example `amount_to_lp` belongs to alice or bob, not to Policy. 

Long story short, you should put this back to how it was.